script/transifex.sh

```diff
@@ -4,21 +4,24 @@
 #
 # Generate and push the source language file to Transifex.
 # This script is called after every successful build on Travis CI.
-# It relies on $TRANSIFEX_PASSWORD being set in .travis.yml
-# The Transifex client must be installed.
 
 if [ "$TRAVIS_REPO_SLUG" == "Electrical-Age/ElectricalAge" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "develop" ]
```

Comment:
@cm0x4D Just realized this one will be hard to sanity check and get green before merging since PR this is not executed for PR :)